src/main/java/frc/robot/subsystems/drivetrain/BaseSwerveSubsystem.java

```diff
@@ -64,6 +69,8 @@ public BaseSwerveSubsystem(
 
         this.kinematics = kinematics;
 
+        SmartDashboard.putData("Field", cfield);
```

Comment:
this will probably conflict with the field already being added in the `apriltag` branch. i think we can just add this feature in `apriltag`

Reply:
field2d already exists in main